CIP-0014/CIP-0014.md

```diff
@@ -0,0 +1,116 @@
+# CIP-0014 User-Facing Asset Id 
+
+# Preamble
+
+---
+CIP: 0014
+Title: User-Facing Asset Id
+Authors: Matthias Benkort <matthias.benkort@iohk.io>, Rodney Lorrimar <rodney.lorrimar@iohk.io>
+Comments-Summary: A common format for displaying a unique identifier for assets.
+Comments-URI: https://github.com/cardano-foundation/CIPs/wiki/Comments:CIP-0014
+Status: Draft
+Type: Standards
+Created: 2020-02-01
+License: CC-BY-4.0
+---
+
+# Abstract
+
+This specification defines a unique user-facing asset id as a bech32-encoded blake2b-256 digest of the concatenation of the policy id and the asset name.
+
+# Motivation
+
+The Mary era of Cardano introduces the support for native assets. On the blockchain, native assets are uniquely identified by both their so-called policy id and asset name. Neither the 
+policy id nor the asset name are intended to be human-readable data. 
+
+On the one hand, the policy id is a hash digest of either a monetary script or a Plutus script. On the other hand, the asset name is an arbitrary bytestring of up to 32 bytes (which does not necessarily decode to a valid UTF-8 sequence). In addition, it is possible for an asset to have an empty asset name, or, for assets to have identical asset names under different policy ids. 
+
+Because assets are manipulated in several user-facing features, and also via hardware devices, it is useful to come up with a short(er) and human-readable identifier for assets that user can recognize and refer to when talking about assets.
+
+# Specification
+
+We define a unique identifier for assets in pseudo-code as:
+
+```
+assetId := encodeBech32
+  ( datapart = hash
+    ( algorithm = 'blake2b'
+    , digest-length = 32
+    , message = policyId | assetName
```

Comment:
for the separator is it possible to use `.`? that way we could match the current documentation here https://developers.cardano.org/en/development-environments/native-tokens/how-to-use-multi-asset-tokens-in-the-cardano-node/

<img width="845" alt="Screen Shot 2021-02-03 at 12 39 01 AM" src="https://user-images.githubusercontent.com/1622112/106824205-f4297180-6647-11eb-8192-add1b8ef2768.png">

Reply:
Note that there's no separator,  `|` designates concatenation. Make it `<>` or `++` if you prefer :sweat_smile:

Reply:
@KtorZ thanks for the explanation -- know the difference but I'm not very pedantic about these things. The main point reminds, if we are going to use an arbitrary symbol to concatenate, we could try to keep the ones that we are already using in other tooling